src/_repobee/cli/mainparser.py

```diff
@@ -141,6 +142,8 @@ def _versioned_plugin_name(plugin_module: types.ModuleType) -> str:
         action="version",
         version="{}".format(_repobee.__version__),
     )
+    # re-add the preparser arguments, though these have already been parsed
+    preparser.add_arguments(parser)
```

Comment:
Could you explain in the comment here _why_ the arguments are added? I.e. that they need to be added in order to show up in the CLI help section.

Reply:
added to the comment.